src/randoop/main/GenTests.java

```diff
@@ -429,264 +399,335 @@ public boolean handle(String[] args) throws RandoopTextuiException {
         }
       }
     }
+    
+    ExecutionVisitor visitor;
+    if (visitors.isEmpty()) {
+      visitor = new DummyVisitor();
+    } else {
+      visitor = new MultiVisitor(visitors);
+    }
+    
+    explorer.addTestPredicate(outputTest);
+    explorer.addTestCheckGenerator(testGen);
+    explorer.addExecutionVisitor(visitor);
 
-    explorer.executionVisitor.visitors.addAll(visitors);
-
+    if (!GenInputsAbstract.noprogressdisplay) {
+      System.out.printf ("Explorer = %s\n", explorer);
+    }
+    
+    //Generate tests
```

Comment:
Was this comment inaccurate or just unnecessary?  As someone not familiar with the Randoop codebase, I found it useful.